app/views/shared/_want_to_talk.html.erb

```diff
@@ -6,21 +6,21 @@
       </span>
     </button>
   </div>
-  <button class="want-to-talk__title">
-    <h2 class="want-to-talk__title__text"><%= t('want_to_talk.title') %></h2>
-  </button>
-  <div class="want-to-talk__body" aria-hidden="true">
-    <p class="want-to-talk__body__intro">
+  <button class="want-to-talk__intro">
+    <h2 class="want-to-talk__intro__title"><%= t('want_to_talk.title') %></h2>
+    <p class="want-to-talk__intro__text">
       <%= t('want_to_talk.intro') %>
     </p>
-    <p class="want-to-talk__body__small-print">
-      <%= t('want_to_talk.small_print') %>
-    </p>
+  </button>
+  <div class="want-to-talk__body" aria-hidden="true">
     <p class="want-to-talk__body__call-us">
       <%= t('want_to_talk.call_us') %>
     </p>
     <a href="<%= t('want_to_talk.phone_number') %>" class="want-to-talk__body__phone">
-      <%= t('want_to_talk.phone_number_readable') %>
+      <%= t('want_to_talk.phone_number_readable') %>*
```

Comment:
For info, Service Delivery said there was no need for Welsh...which I
challenged, but their call!
On 4 Feb 2015 11:48, "Ben Barnett" <notifications@github.com> wrote:

> In app/views/shared/_want_to_talk.html.erb
> <https://github.com/moneyadviceservice/frontend/pull/893#discussion_r24077902>
> :
>
> >      <p class="want-to-talk__body__call-us">
> >        <%= t('want_to_talk.call_us') %>
> >      </p>
> >      <a href="<%= t('want_to_talk.phone_number') %>" class="want-to-talk__body__phone">
> > -      <%= t('want_to_talk.phone_number_readable') %>
> > +      <%= t('want_to_talk.phone_number_readable') %>*
>
> Oops - sorry ignore, I need to look up!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/moneyadviceservice/frontend/pull/893/files#r24077902>.
>